pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -68,20 +62,16 @@
 
     LGPU_CPP_BINARY_AVAILABLE = True
 
-    try:
-        from ._mpi_handler import MPIHandler
-
-        MPI_SUPPORT = True
-    except ImportError as ex:
-        warn(str(ex), UserWarning)
-        MPI_SUPPORT = False
-
-    LGPU_CPP_BINARY_AVAILABLE = True
 except (ImportError, ValueError) as ex:
     warn(str(ex), UserWarning)
     LGPU_CPP_BINARY_AVAILABLE = False
     backend_info = None
 
+from ._adjoint_jacobian import LightningGPUAdjointJacobian
+from ._measurements import LightningGPUMeasurements
+from ._mpi_handler import MPIHandler
+from ._state_vector import LightningGPUStateVector
+
```

Comment:
Move the import of the libraries after the first import `pennylane_lightning.lightning_gpu_ops`